LogControl/Infrastructure/Messaging/LogMessageConsumer.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Contracts.Logs.DTOs;
+using LogControl.Application.Interfaces;
+using LogControl.Domain.Entity;
+using MassTransit;
+
+namespace LogControl.Infrastructure.Messaging
+{
+    public class LogMessageConsumer : IConsumer<LogMessageDto>
+    {
+        private readonly ILogService _logService;
+
+        public LogMessageConsumer(ILogService logService)
+        {
+            _logService = logService;
+        }
+
+        public async Task Consume(ConsumeContext<LogMessageDto> context)
```

Comment:
I would say that it's risky to have a separate microservice for logs because if it falis/rabbitmq fails/ you'll never get error and wouldn't be able to know the errors so it's better not to use rabbit mq for logs and handle logs on the level of each microservice separately (and this is a usual practice). For pet project it's fine but for real cases it's better to user rabbit mq for some events handling/db consistency if redundant, etc.